### ISSUE TITLE
Want more details of training KITTI Distance

### ISSUE THREAD

**Issue by Tommy-yours** [luogen1996/RepAdapter#4]
Hi, recently I reproduce the performance of AdaptFormer. However, I only get 50% acc on the KITTI Distance of VTAB while other results are normal, and I wonder if there are some mistakes in my train script. So, if convenience, I want to know your script of training KITTI Distance to improve my script, including learning rate, weight decay and so on. Also, if you can provide the script of training KITTI Distance in AdaptFormer, it will be better. Looking forward to your reply. Thanks a lot.

**Comment by luogen1996** [comment]
> Hi, recently I reproduce the performance of AdaptFormer. However, I only get 50% acc on the KITTI Distance of VTAB while other results are normal, and I wonder if there are some mistakes in my train script. So, if convenience, I want to know your script of training KITTI Distance to improve my script, including learning rate, weight decay and so on. Also, if you can provide the script of training KITTI Distance in AdaptFormer, it will be better. Looking forward to your reply. Thanks a lot.

Thanks for your interests in our work. This repo does not provide the implementation of AdaptFormer. If you want to reproduce the results of AdaptFormer, please refer to thier official repo.

**Comment by Tommy-yours** [comment]
okay, thank you , I will close.
